rtl/eclrtl/rtlrecord.cpp

```diff
@@ -175,20 +179,30 @@ static unsigned expandNestedRows(unsigned idx, unsigned startIdx, const char *pr
             {
                 StringBuffer newPrefix(prefix);
                 if (cur->name && *cur->name)
-                    newPrefix.append(cur->name).append('.');
-                idx = expandNestedRows(idx, isIfBlock ? startIdx : idx, newPrefix.str(), nested, target, names, nestIfBlock, ifblocks);
+                    newPrefix.append(cur->name);
+                idx = expandNestedRows(idx, isIfBlock ? startIdx : idx, newPrefix.str(), nested, target, names, xpaths, nestIfBlock, ifblocks);
             }
         }
         else
         {
             if (prefix)
             {
                 StringBuffer name(prefix);
-                name.append(cur->name);
+                name.append(cur->name).append('.');
```

Comment:
This looks wrong - I would have expected '.' then the name.
You also can't use the same prefix for xpaths and names because if they nest more deeply you'll have a.b/c for the xpath.
(Note this really does happen 3 or 4 levels deep)

Reply:
whoops, yes obvious now you point it out :)
Will fix.